### PR TITLE
sched/cpuload: Define function to clear cpuload count of defunct thread

### PR DESCRIPTION
**Pull request by jeongarmy**
Define function to clear cpuload count of defunct thread and call it when releasing pid

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [9914eab737318d31e383b459f45339be3fd6bf76] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [9914eab737318d31e383b459f45339be3fd6bf76] - Code Rule Check OK.

File: os/kernel/sched/sched.h [hunk 1]
```diff
@@ -272,9 +264,12 @@ void sched_timer_reassess(void);
 #define sched_timer_reassess()
 #endif
 
-#if defined(CONFIG_SCHED_CPULOAD) && !defined(CONFIG_SCHED_CPULOAD_EXTCLK)
+#ifdef CONFIG_SCHED_CPULOAD
+#ifndef CONFIG_SCHED_CPULOAD_EXTCLK
 void weak_function sched_process_cpuload(void);
 #endif
+void sched_clear_cpuload_pid(pid_t pid);
```
Comment (sunghan-chang) [inline os/kernel/sched/sched.h]: 1. When `CONFIG_SCHED_CPULOAD_EXTCLK` is enabled, is this necessary?
2. Why is `sched_process_cpuload()` in `CONFIG_SCHED_CPULOAD_EXTCLK` conditional even its function is not in that conditional?

**Comment by seinfra** [comment]
Target : [38f2f4a8633dda65a931684434c029eb2cd40de1] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [38f2f4a8633dda65a931684434c029eb2cd40de1] - Code Rule Check (C++) OK.

**Pull request merged**